app/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionController.scala

```diff
@@ -41,48 +39,26 @@ class IntroductionController @Inject() (val authConnector: AuthConnector,
   extends BaseController with HelpToSaveAuth {
 
   def getAboutHelpToSave: Action[AnyContent] = unprotected { implicit request ⇒ implicit htsContext ⇒
-    handleIntroductionPage(
-      views.html.introduction.about_help_to_save(),
-      "https://www.gov.uk/get-help-savings-low-income"
-    )
+    SeeOther("https://www.gov.uk/get-help-savings-low-income")
```

Comment:
it'd be more flexible for the future to read the beginning part of the URL from config (`govuk.urls.base`)

Reply:
fixed